src/services/emitter.js

```diff
@@ -20,6 +20,6 @@
  *
  */
 
-import Vue from 'vue'
+import mitt from 'mitt'
 
-export const EventBus = new Vue()
+export const emitter = mitt()
```

Comment:
I think we should keep the name `EventBus`. Although it has a different implementation internally now, it still represent the same thing - the event bus / event hub, not any a general emitter.

Reply:
I changed the name to reflect the inner implementation change, and to if EventBus remained, `.once` can be mistakenly used.

Reply:
but will revert the naming as I will define `once` method

Reply:
> I changed the name to reflect the inner implementation change,

But it should not reflect inner implementation, it should be behind the scenes via "public API".